### PR TITLE
WT-4625 Add a check for camelCase names

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

**Comment by lukech** [comment]
autobuild: Can one of the admins verify this patch?

File: dist/s_style [hunk 1]
```diff
@@ -23,6 +23,11 @@ if [ $# -ne 1 ]; then
 	    -e '/checksum\/power8/d' \
 	    -e '/checksum\/zseries/d' |
 	xargs $xp -n 1 -I{} sh ./dist/s_style {}
+
+	# Checks source files for camel case formatting, variables should be separated by underscore instead.
+	if egrep -n -r '\b[a-z]+[A-Z]' src | egrep -v '/os_win/|/docs/|/tags|:[     ]+\*|"|hash_city|huffman|checksum|UNCHECKED_STRING'; then
```
Comment (sulabhM) [inline dist/s_style:28]: @mm-ng , I would suggest we follow the existing way of running the tests per file. So basically moving the code in the else portion and checking for whether to exclude the files in there.

**Comment by sulabhM** [comment]
PR testing has hit known bug: https://jira.mongodb.org/browse/WT-6800

File: dist/s_style [hunk 2]
```diff
@@ -23,6 +23,7 @@ if [ $# -ne 1 ]; then
 	    -e '/checksum\/power8/d' \
 	    -e '/checksum\/zseries/d' |
 	xargs $xp -n 1 -I{} sh ./dist/s_style {}
+
```
Comment (sulabhM) [inline dist/s_style:26]: Please get rid of redundant empty line

**Review by sulabhM — APPROVED** [review]
LGTM mod removing a redundant new line

**Pull request merged**